dynamic-filter-script.js

```diff
@@ -0,0 +1,45 @@
+(async () => {
+  var url = document.URL;
+  const regex = /https:\/\/www\.drupal\.org\/project\/issues\/.*/g;
+  if (url.match(regex)) {
+    const issueTable = document.querySelector("table.project-issue");
```

Comment:
I just pushed some changes to `main` branch that relate to this.

To try to not duplicate this logic everywhere I changed `drupal-content-queue.js` to be the main file loaded on the issue queue listing page. so that file would be responsible for determining if we on the search page and making a `custom-toolbar` element where the other content js files would add their elements.

See how I import the existing `drupal-user-count.js` at the top of `drupal-content-queue.js`

So to update this PR to match the new pattern you would need to 
1. add `dynamic-filter-script.js` to `web_accessible_resources` in `manifest.json` instead of `content_scripts`
2. change `dynamic-filter-script.js` to declare a function at adds the searchInput passed in `customToolbar` element. see the new `drupal-user-count.js` 
3. import `dynamic-filter-script.js` at the top of `drupal-content-queue.js` see the existing 

  ```
src = chrome.runtime.getURL("drupal-user-count.js");
  const { userCount } = await import(src);
```
4. in `drupal-content-queue.js`  call the new function like the existing `customToolbar.appendChild(userCount.createElement())`